### DISCUSSION TITLE
Course schedule

### DISCUSSION THREAD

**Discussion by tmgoodman** [giswqs/geog-312#36] [Q&A] [answered]
Hi! I am curious about the time commitment for this course, specifically when the midterm exam will be and the timeline of the final project. I am trying to determine if this course will fit with my school schedule!

**Comment by giswqs** [comment, marked as answer]
It is a self-paced course. There are no assignments unless you are enrolled in this course. There are no quizzes or midterm for non-UTK students. 

https://geog-312.gishub.org/#enroll-for-certification
